roles/branch-validation/tasks/main.yml

```diff
@@ -155,7 +167,7 @@
     that: ping.user_input|bool
 
 - include: ha.yml
-  when: validate_ha_environment
+  when: not groups['compute'] | intersect(groups['controller']) | length | bool
```

Comment:
So here's my problem. We need to test HA whether or not the cluster has converged controllers. We still have to test that having one controller go down does not disrupt the data path for the instances in the cluster.

It's just that if the cluster is not converged, we shouldn't worry about the status of instances on the downed controller coming back, because there are no instances on the downed controller.

Does that make sense?